acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -87,7 +87,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH="#{agent[:privatebindir]}:$PATH" curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
Since we're in a HEREDOC, variable interpolation should already be occurring. Is there a reason this change is needed for cisco? I'm concerned it may break other platforms, as quoting can be tricky with beaker, net-ssh and the remote side.